Georeferencing-Tin-12d-3/Georeferencing-Tin-12d-3.ifc

```diff
@@ -0,0 +1,43 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION (('ViewDefinition[Ifc4x3NotAssigned]'), '2;1');
+FILE_NAME ('Georeferencing-Tin-12d-3.ifc', '2021-04-01T00:59:35', ('Lee Gregory'), ('12d Solutions'), 'IFC Engine rev 1257', '12d Model 15.0 Alpha 553 [DEVELOP] Macro exit codes', '');
+FILE_SCHEMA (('IFC4X3_RC3'));
+ENDSEC;
+DATA;
+#1 = IFCPROJECT('19S2rVW_v6h9wDt$Ub99R0', #2, 'Georeferencing-Tin-3', 'Tin with hole MGA94 Zone 56', $, $, $, (#14), #9);
+#2 = IFCOWNERHISTORY(#3, #8, $, .NOCHANGE., $, $, $, 1617199175);
+#3 = IFCPERSONANDORGANIZATION(#4, #6, $);
+#4 = IFCPERSON($, 'Gregory', 'Lee', $, $, $, (#5), $);
+#5 = IFCACTORROLE(.CIVILENGINEER., 'Designer', $);
+#6 = IFCORGANIZATION($, '12d Solutions', $, (#7), $);
+#7 = IFCACTORROLE(.CONSULTANT., 'Engineering', $);
+#8 = IFCAPPLICATION(#6, '15.0 Alpha 553 [DEVELOP] Macro exit codes', '12d Model', '12d Model');
+#9 = IFCUNITASSIGNMENT((#10, #11, #12, #13));
+#10 = IFCSIUNIT(*, .LENGTHUNIT., $, .METRE.);
+#11 = IFCSIUNIT(*, .AREAUNIT., $, .SQUARE_METRE.);
+#12 = IFCSIUNIT(*, .VOLUMEUNIT., $, .CUBIC_METRE.);
+#13 = IFCSIUNIT(*, .PLANEANGLEUNIT., $, .RADIAN.);
+#14 = IFCGEOMETRICREPRESENTATIONCONTEXT($, 'Model', 3, 1.E-6, #16, #20);
+#15 = IFCLOCALPLACEMENT($, #16);
+#16 = IFCAXIS2PLACEMENT3D(#17, #18, #19);
+#17 = IFCCARTESIANPOINT((0., 0., 0.));
+#18 = IFCDIRECTION((0., 0., 1.));
+#19 = IFCDIRECTION((1., 0., 0.));
+#20 = IFCDIRECTION((0., 1.));
+#21 = IFCPROJECTEDCRS('EPSG:7856', 'MGA2020 Zone 56 and AHD', 'EPSG:7844', 'EPSG:5711', 'UTM', '56', $);
+#22 = IFCMAPCONVERSION(#14, #21, 0., 0., 0., 1., 0., 1., 1., 1.);
+#23 = IFCSITE('1n$l6IimP2IhA9iA5hol56', #2, 'Default Site', 'Description of Default Site', $, #15, $, $, .ELEMENT., $, $, $, $, $);
+#24 = IFCRELAGGREGATES('13faDwIv10yPVop6NVH4yU', #2, 'ProjectContainer', 'ProjectContainer for Sites', #1, (#23));
+#25 = IFCFACILITY('2C6f2w14v109DFWVHnhc31', #2, 'tin tiny with hole', 'Description of tin tiny with hole', $, #15, $, $, .ELEMENT.);
```

Comment:
As with #84, the Terrain is contained in a facility rather than the site (the more logical container) and is not needed unless further managed or built elements are included